packages/esm-active-visits-app/src/visits-summary/visits-components/medications-summary.component.tsx

```diff
@@ -16,36 +16,83 @@ const MedicationSummary: React.FC<MedicationSummaryProps> = ({ medications }) =>
+                        )}
+                        {medication?.order?.frequency?.display?.toLowerCase()} &mdash;{' '}
+                        {!medication?.order?.duration
+                          ? t('orderIndefiniteDuration', 'Indefinite duration')
+                          : t('orderDurationAndUnit', 'for {{duration}} {{durationUnit}}', {
+                              duration: medication?.order?.duration,
+                              durationUnit: medication?.order?.durationUnits?.display?.toLowerCase(),
+                            })}
+                        {medication?.order?.numRefills !== 0 && (
+                          <span>
+                            <span className={styles.label01}> &mdash; {t('refills', 'Refills').toUpperCase()}</span>{' '}
+                            {medication?.order?.numRefills}
+                            {''}
+                          </span>
+                        )}
+                        {medication?.order?.dosingInstructions && (
+                          <span> &mdash; {medication?.order?.dosingInstructions?.toLocaleLowerCase()}</span>
+                        )}
+                      </p>
+                      <p className={styles.bodyLong01}>
+                        {medication?.order?.orderReasonNonCoded ? (
+                          <span>
+                            <span className={styles.label01}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
+                            {medication?.order?.orderReasonNonCoded}
+                          </span>
+                        ) : null}
+                        {medication?.order?.quantity ? (
+                          <span>
+                            <span className={styles.label01}> &mdash; {t('quantity', 'Quantity').toUpperCase()}</span>{' '}
+                            {medication?.order?.quantity}
+                          </span>
+                        ) : null}
+                        {medication?.order?.dateStopped ? (
+                          <span className={styles.bodyShort01}>
+                            <span className={styles.label01}>
+                              {medication?.order?.quantity ? ` — ` : ''} {t('endDate', 'End date').toUpperCase()}
+                            </span>{' '}
+                            {formatDate(new Date(medication?.order?.dateStopped))}
+                          </span>
+                        ) : null}
+                      </p>
+                    </div>
+                  </div>
+
+                  <p className={styles.metadata}>
+                    {formatTime(parseDate(medication?.order?.dateActivated))}
+                    {medication?.provider?.name && <> &middot; {medication?.provider?.name}</>}
+                    {medication?.provider?.role && <>, {medication?.provider?.role}</>}
+                  </p>
+                </React.Fragment>
+              ),
+          )}
+        </div>
```

Comment:
I created a common component for this, let me find it

Reply:
MY bad, I thought I did create it

Reply:
Yeah, it's not great that the code in the Medications and Notes summary is basically a copy of what exists in the Patient Chart. Maybe it makes more sense to expose those components as extensions and then leverage them here similar to what we're doing for the Results Viewer widget in the Tests summary, maybe not @brandones.